python/nano/src/bigdl/nano/utils/inference/tf/model.py

```diff
@@ -41,7 +42,7 @@ def __call__(self, *args, **kwds):
         return super().__call__(*args, **kwds)
 
     def call(self, *inputs):
-        return tf.py_function(self.forward, inputs, Tout=tf.float32)
+        return tf.py_function(self.forward, inputs, Tout=self.precision)
```

Comment:
I'm not sure whether it's necessary, the output of `mixed_bfloat16` may also be float32 ?

Reply:
yeah, you are right. We currently make all layers(except the input layer) bf16, we may add a parameter (need design) later for customized requirement (like if the last layer needs to be fp32...), so that we could change our `Tout`.